build.zig.zon

```diff
@@ -0,0 +1,23 @@
+.{
+    .name = .zig_enr,
+
+    .version = "0.0.0",
+
+    .fingerprint = 0x499ae5d25c594f6a,
+
+    .minimum_zig_version = "0.14.1",
+
+    .dependencies = .{
+        .secp256k1 = .{
+            .url = "git+https://github.com/zig-bitcoin/libsecp256k1-zig#b96922b375a601f3303bf43be1dceb3d101df6c6",
```

Comment:
does this solve the signature mismatch issue?

Reply:
yes, right now it can generate same signature with other implementations